stable/spinnaker/values.yaml

```diff
@@ -71,8 +72,24 @@ deck:
     #    hosts:
     #      - domain.com
 
+front50:
```

Comment:
in this chart version the resources are set the same for all charts. Can you leverage those values to set the -Xmx?